lib/actions-oclif/build.ts

```diff
@@ -139,21 +142,6 @@ ${dockerignoreHelp}
 		logger.logSuccess('Build succeeded!');
 	}
 
-	protected translateParams(params: ArgsDef, options: FlagsDef) {
-		// Copy flags to those expected by other modules
-		options.arg = options.buildArg;
-		delete options.buildArg;
-		options['image-list'] = options['cache-from'];
-		delete options['cache-from'];
```

Comment:
@srlowe, FYI, I guess that this "mapping" of `buildArg` to `arg` and from `cache-from` to `image-list` originated from a misunderstanding of Capitano's `signature` and `parameter` properties:

https://github.com/balena-io/balena-cli/blob/v12.9.8/lib/utils/docker-js.js#L83-L84
```
			signature: 'buildArg',
			parameter: 'arg',

			signature: 'cache-from',
			parameter: 'image-list',
```
The `parameter` property is not the "new name" of the flag, but merely what Capitano prints in the help output as follows:
```
$ ./balena-cli-12.9.8/balena help build
...
    --buildArg, -B <arg>
...
    --cache-from <image-list>
```
The flags are still referred to in the code as `options.buildArg` and `options['cache-from']`, despite the `parameter` property.